controllers/account.js

```diff
@@ -6,27 +6,27 @@ const authHelper = require('../helpers/authentication');
 // secure routes
 router.use(authHelper.authChecker);
 
-router.post('/', function (req, res, next) {
-    const { firstName, lastName, email, password, password_new } = req.body; // TODO: sanitize
-        return api(req).patch('/accounts/' + res.locals.currentPayload.accountId, {
+router.post('/', function (req, res) {
```

Comment:
Just a suggestion:
I would be even nicer to use `await` instead of all the `then`s

Reply:
I agree with that, but I'm not sure if we can use `await` without breaking anything, because the dependencies are quit old and I'm not sure if we may need a compiler like Babel.js. I also didn't want to break with the general coding style of the repository.